api_testing/apis/orders_api.py

```diff
@@ -292,6 +297,10 @@ def create_order_with_double_test_plans(self, only_test_plans=False):
         testplan['id'] = testplan_form_data['testPlan']['testPlanEntity']['id']
         article = testplan_form_data['testPlan']['selectedArticles'][0]['name']
         article_id = testplan_form_data['testPlan']['selectedArticles'][0]['id']
+        if article == 'all':
+            selected_article = random.choice(ArticleAPI().get_all_articles(limit=10)[0]['articles'])
+            article = selected_article['name']
+            article_id = selected_article['id']
```

Comment:
same code has been coded twice >> then it should be abstract as a function and call it in those two locations

Reply:
done